arm-containerregistry/2017-06-01-preview/swagger/containerregistry.json

```diff
@@ -732,13 +741,20 @@
             "schema": {
               "$ref": "#/definitions/Webhook"
             }
+          },
+          "201": {
+            "description": "The request was successful; the operation will complete asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/Webhook"
+            }
           }
         },
         "x-ms-examples": {
           "WebhookUpdate": {
             "$ref": "../examples/WebhookUpdate.json"
           }
-        }
+        },
+        "x-ms-long-running-operation": true
```

Comment:
This api-version hasn't gone alive yet so any breaking change should be acceptable. This change is requested by ARM team.